source/releasenotes/2024-05-02-Synthetics-Check-Limits.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "New Relic Synthetics: New monthly Check limits Starting May 1st, 2024"
+published_date: "2024-05-02"
+categories: [action-required]
+---
+**New Relic Synthetics: New monthly Check limits**
+
+Beginning on May 1st, 2024, Pantheon will begin limiting the number of Synthetics checks for each Pantheon account.
```

Comment:
May 1st is in the past now - could we update to past tense?

Reply:
Updated.